### ISSUE TITLE
Changes to eahub.org/group/* text

### ISSUE THREAD

**Issue by cafelow** [rtcharity/eahub.org#927]
Instead of "Status: Inactive"
write "Status: This group is currently not meeting regularly. You can still connect with EAs in the area through the links below."

**Comment by sebgrebe** [comment]
@cafelow 2 questions:
1) I'm worried that this might be a bit wordy and breaks with the style of information we otherwise have in the keys facts box (usually just one-word pieces of information like "City" or an email address)
![Screen Shot 2020-05-26 at 12 06 28](https://user-images.githubusercontent.com/15908208/82893813-84f06f00-9f49-11ea-98c8-280c9b5ae1be.png)
2) At least given how out database is set up, it's not guaranteed that each group will have some links below the status field. It's possible to only have a name for a group, without any emails or websites. So the status could refer to links that don't exist

**Comment by cafelow** [comment]
1. I see what you mean. But I still want the information to get across somehow. This is a little shorter: 

> Status: This group is inactive. You can still connect with EAs in the area through the links below. 

2. I'm planning on hiding all groups that don't have any contact information, so you don't have to worry about that.
